docs/release-notes/flink-1.10.md

```diff
@@ -73,16 +73,56 @@ start on YARN. This option has been deprecated since the introduction of
 [FLIP-6](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65147077).
 All Flink users are advised to remove this command line option.
 
+#### Removal of --yst/--yarnstreaming Command Line Options ([FLINK-14957](https://issues.apache.org/jira/browse/FLINK-14957))
+The Flink CLI no longer supports the deprecated command line options
+`-yst/--yarnstreaming`, which were used to disable eager pre-allocation of memory.
+All Flink users are advised to remove this command line option.
+
 #### Mesos Integration will reject expired Offers faster ([FLINK-14029](https://issues.apache.org/jira/browse/FLINK-14029))
 Flink's Mesos integration now rejects all expired offers instead of only 4.
 This improves the situation where Fenzo holds on to a lot of expired offers
 without giving them back to the Mesos resource manager.
 
 #### Scheduler Rearchitecture ([FLINK-14651](https://issues.apache.org/jira/browse/FLINK-14651))
 Flink's scheduler was refactored with the goal of making scheduling strategies
-customizable in the future. Users that experience issues related to scheduling
-can fallback to the legacy scheduler by setting `jobmanager.scheduler: legacy`
-in their `flink-conf.yaml`.
+customizable in the future. Users that did not already use the [Pipelined
+Region Failover Strategy]({{ site.baseurl
+}}/dev/task_failure_recovery.html#restart-pipelined-region-failover-strategy)
+in Flink 1.9, should take the following caveats into account:
+
+* Exceptions that caused a job to restart will not shown on the job overview page of the Web UI ([FLINK-15917](https://issues.apache.org/jira/browse/FLINK-15917)).
+However, exceptions that cause a job to fail (e.g., when all restart attempts exhausted) will still be shown.
+* The `uptime` metric will not be reset after restarting a job due to task failure ([FLINK-15918](https://issues.apache.org/jira/browse/FLINK-15918)).
```

Comment:
Should be fixed with #11032.

Reply:
However, we should say that this is the case when using the legacy scheduler combined with regional failover.